src/main/webapp/gallery.html

```diff
@@ -3,17 +3,18 @@
     <head>
         <meta charset="UTF-8">
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-        <title>Gallery</title>
+        <title>Pieced - Gallery</title>
 
         <link rel="stylesheet" href="gallery.css">
         <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+
     </head>
 
     <body>
         <!-- Navigation -->
```

Comment:
I think we could replace this with the navbar that Fiza was working on as a next step

Reply:
Yep I'll change that once we merge the navbar!